docs/diagrams/BetterModelClassDiagram.puml

```diff
@@ -4,24 +4,40 @@ skinparam arrowThickness 1.1
 skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
-AddressBook *-right-> "1" UniquePersonList
```

Comment:
I think would be good to leave the UniquePersonList in

Reply:
Ok